app/bundles/CoreBundle/Assets/css/libraries/libraries.css

```diff
@@ -1,4 +1,5 @@
 /*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+@import "../../../../../../node_modules/remixicon/fonts/remixicon.css";
```

Comment:
Untested, but I think this could be changed to just `remixicon/fonts/remixicon`? this way it is more robust against path changes and in the same style as other imports.

Reply:
I just tested this, it works and it's a better solution. The current solution does not work well in prod mode after running `mautic:assets:generate`. The styles is `remixicon.css` do not get expanded into the `libraries.css` file when the generates.

Reply:
Actually, I take that back. When importing with `remixicon/fonts/remixicon` instead of with `../../../../../../node_modules/remixicon/fonts/remixicon.css` it wrote the css into the `libraries.css` but for some reason the browser didn't apply the styles for the "Settings" icon.

I ended up reverting the `@import` back and moved the `node_modules` directory into my `docroot` so that it could resolve the import.